oioswift/common/middleware/copy.py

```diff
@@ -261,9 +261,8 @@ def __call__(self, env, start_response):
         # FIXME: handle COPY method (with Destination-* headers)
         if self.fast_copy_allowed(req):
             self.logger.debug("COPY: fast copy allowed")
-            # FIXME(FVE): we may use req.environ instead of headers
-            req.headers['Oio-Copy-From'] = req.headers.get('X-Copy-From')
-            del req.headers['X-Copy-From']
+            env['HTTP_OIO_COPY_FROM'] = unquote(env['HTTP_X_COPY_FROM'])
```

Comment:
This is not what I meant with the FIXME. This is still using a header. I was thinking of a key like `'oio.copy_from'`.